src/fmt.rs

```diff
@@ -246,6 +246,12 @@ mod test {
         let input = fs::read_to_string("examp/right.toml").unwrap();
         let mut toml = input.parse::<Document>().unwrap();
         fmt_toml(&mut toml, &Config::new());
+        #[cfg(target_os = "windows")]
+        assert_eq!(
+            input.replace("\r\n", "\n"),
+            toml.to_string_in_original_order().replace("\r\n", "\n")
```

Comment:
It's been a while since I've worked on this so I could be way off but, could you remove the `replace` call since the commit after this should fix this?

Reply:
I played with it a bit more and it looks like if the `.gitattributes` flag for `toml` files is set to `lf` we can remove the `replace` calls.
```
// .gitattributes file 
# Set the default behavior, in case people don't have core.autocrlf set.
* text=auto

*.toml text eol=lf
```

It's either: git does the replacing or we do in the tests. 
Let me know what you prefer.

Reply:
Ehh yeah let's just leave it as is, and again thanks for everything!